### PR TITLE
[INFINITY-1653]template framework overlay tests

### PR DESCRIPTION
**Pull request by ArtRand**
Tests configurable overlay support in the template framework.

### CODE REVIEW

**Comment by mbdcos** [comment]
LGTM

**Comment by ArtRand** [comment]
retest this please (marathon)

**Comment by ArtRand** [comment]
retest this please (retry marathon problem?)

File: testing/sdk_networks.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+                    assert "name" in network_info, \
+                        "Didn't find network name in NetworkInfo for task {task} with " \
+                        "status:{status}".format(task=task_name, status=status)
+                    assert network_info["name"] == expected_network_name, \
+                        "Expected network name:{expected} found:{observed}" \
+                        .format(expected=expected_network_name, observed=network_info["name"])
+                else:
+                    assert "name" not in network_info, \
+                        "Task {task} has network name when it shouldn't has status:{status}" \
+                        .format(task=task_name, status=status)
```
Comment (susanxhuynh) [inline testing/sdk_networks.py]: No newline at end of file.

**Pull request merged**